source/configuration/modules/omlibdbi.rst

```diff
@@ -113,16 +211,22 @@ it received limited cross-platform tests. If you run into troubles, be
 sure the let us know at
 `http://www.rsyslog.com <http://www.rsyslog.com>`_.
 
-**Sample:**
+
+Examples
+========
+
+Example 1
+---------
 
 The following sample writes all syslog messages to the database
 "syslog\_db" on mysqlserver.example.com. The server is MySQL and being
 accessed under the account of "user" with password "pwd" (if you have
 empty passwords, just remove the $ActionLibdbiPassword line).
 
-::
+.. code-block:: none
+
+   $ModLoad omlibdbi $ActionLibdbiDriver mysql $ActionLibdbiHost
```

Comment:
Suggestion: If the [`advanced`](http://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html) configuration format is supported, convert the example to use that format.